tests/ui/panic-runtime/need-immediate-abort-got-abort.rs

```diff
@@ -2,7 +2,7 @@
 //@ aux-build:needs-immediate-abort.rs
 //@ compile-flags:-C panic=abort
 //@ no-prefer-dynamic
-//@ add-core-stubs
+//@ add-minicore
 //@ core-stubs-compile-flags: -Zunstable-options -Cpanic=immediate-abort
```

Comment:
Problem: I realized that Saethlin recently added `//@ core-stubs-compile-flags` for overriding the flags used to build minicore. We should also rename this, but I'll do it in a follow-up, tracked in #148360.